src/main/update/LonghornProvider.ts

```diff
@@ -18,6 +18,10 @@ export interface LonghornUpdateInfo extends UpdateInfo {
   requestIntervalInMinutes: number;
 }
 
+export function isLonghornUpdateInfo(info: UpdateInfo): info is LonghornUpdateInfo {
```

Comment:
I don't see why the return type here can't be a simpler `Boolean` because in the one call to `isLonghornUpdateInfo` there's no further type propagation for typescript to use.